doc/source/conf.py

```diff
@@ -41,6 +41,11 @@
 import sunpy.data
 sunpy.data.download_sample_data(overwrite=False)
 
+# -- Shut up numpy warnings from WCSAxes
+
+import numpy as np
+np.seterr(invalid='ignore')
```

Comment:
ok so it seems that the pandas warnings are not actually 3.6 they also happen on 3.5 so back to the drawing board on that one.